scalardb/test/scalardb/core_test.clj

```diff
@@ -25,7 +25,9 @@
       (condp = column
         "tx_id" (Optional/of (TextValue. column id))
         "tx_created_at" (Optional/of (BigIntValue. column (long 1566376246)))
-        "tx_state" (Optional/of (IntValue. column (Integer/parseInt id)))))))
+        "tx_state" (Optional/of (IntValue. column (Integer/parseInt id)))
+        ;; for the coordinator table
```

Comment:
This is unrelated to the bugfix.
For The group commit feature in the latest ScalarDB.